test/operator-pick.spec.ts

```diff
@@ -87,6 +87,15 @@ describe('pick operator unit tests', () => {
     expect(output![0].contact.email).to.eq(data.contact.email);
   });
 
+  it('it should find properties at different depths', () => {
```

Comment:
Similar needs: decide on whether to use CEDDL mock or local test data, and add `favorites` to pick list.